packages/react/src/components/Switch/Switch.js

```diff
@@ -32,9 +43,17 @@ const Switch = props => {
   const handleKeyDown = e => {
     const key = e.key || e.which;
 
-    if (key === 'Enter' || key === 13 || key === ' ' || key === 32) {
+    if (
+      match(key, keys.SPACE || keys.ENTER || keyCodes.SPACE || keyCodes.ENTER)
```

Comment:
this is always matching `keys.SPACE`. so right now this is not matching enter or space bar either

you need to make the comparisons separately

```suggestion
      match(key, keys.SPACE) ||
      match(key, keys.ENTER) ||
      match(key, keyCodes.SPACE) ||
      match(key, keyCodes.ENTER)
```

Reply:
Oh great catch!